### ISSUE TITLE
Make server latency information available to screen readers

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#6665]
When a server list item has focus, the latency score for that location should be conveyed to screen readers.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-4603)
┆Reporter: Lesley Norton

**Comment by data-sync-user** [comment]
➤ Lesley Norton commented:

Closing as dupe

**Comment by data-sync-user** [comment]
➤ Santiago Andrigo commented:

Moving back to Cancelled
